pkg/cloud/services/eks/nodegroup.go

```diff
@@ -253,33 +301,22 @@ func (s *NodegroupService) deleteNodegroupAndWait() (reterr error) {
 	return nil
 }
 
-func (s *NodegroupService) reconcileNodegroupVersion(ng *eks.Nodegroup) error {
-	var specVersion *version.Version
-	if s.scope.Version() != nil {
-		specVersion = parseEKSVersion(*s.scope.Version())
-	}
-	ngVersion := version.MustParseGeneric(*ng.Version)
-	specAMI := s.scope.ManagedMachinePool.Spec.AMIVersion
-	ngAMI := *ng.ReleaseVersion
-
+func (s *NodegroupService) reconcileNodegroupVersion(ng *awseks.Nodegroup) error {
 	eksClusterName := s.scope.KubernetesClusterName()
-	if (specVersion != nil && ngVersion.LessThan(specVersion)) || (specAMI != nil && *specAMI != ngAMI) {
-		input := &eks.UpdateNodegroupVersionInput{
+	ltVersion := s.scope.ManagedMachinePool.Status.LaunchTemplateVersion
```

Comment:
this now means it's possible to jump from more than one version e.g. `1.16` to `1.18`, even if you don't specify an AMI

Reply:
Am i correct in thinking its only on the control plane that you have to upgrade in increments of 1 ?

Reply:
Yeah you are actually, I can't find anything about doing increments of 1, not sure where I got that stuck in my head from.